src/build/config/config.yaml

```diff
@@ -2101,6 +2101,12 @@ option:
       - auto
       - web-id
 
+  repo-s3-kms-key-id:
+    inherit: repo-s3-bucket
+    required: false
+    deprecate:
+      repo-s3-kms-key-id: {}
```

Comment:
This is a new option so no deprecation is needed.

Reply:
TBH this just mirrors all the other repo-s3 options. Looks like a lot of cargo-cult in there.

I'll remove it for this option.

Reply:
> Looks like a lot of cargo-cult in there.

These deprecations have purposes and all have tests. The option it looks like you copied from (`repo-s3-region`) was available pre-2.0 before indexes were required. So this deprecation allows `repo-s3-region` and makes it equal to `repo1-s3-region`.

If you identify any dead code in this file we'd be happy to remove it.

Reply:
Ok. That's really not visible from "repo-s3-region: deprecate: repo-s3-region", sorry for not grasping that.